### ISSUE TITLE
Commas keep being appended for this array configuration.

### ISSUE THREAD

**Issue by RabugenTom** [nicklockwood/SwiftFormat#23]
For this configuration, a comma is appended after each run of SwiftFormat (v0.9):
```swift
let A = [
    x, // Some comment
]
```
becomes after a few runs:
```swift
let A = [
    x, // Some comment,,,
]
```
If the array have several rows, it only affects the last one.

**Comment by nicklockwood** [comment]
Thanks, I'll get that fixed.

**Comment by nicklockwood** [comment]
Fixed in version 0.9.1.

It won't remove the commas it's already added I'm afraid, but it shouldn't add any more.
